torch/csrc/jit/tracer_state.h

```diff
@@ -60,7 +60,7 @@ struct TracingState : public std::enable_shared_from_this<TracingState> {
   std::unordered_map<void*, Value*> buffer_map;
   // A pair of (input_flags, output_flags) for each stage
   io_variable_flags_list var_flags;
-  std::vector<function_list> output_edges;
+  std::vector<std::vector<autograd::Edge>> output_edges;
```

Comment:
`edge_list`?

Reply:
Ok. Btw, is there any reason why all the `typedef`s are `snake_case` while all our classes are `CamelCase`? E.g. `function_list` vs. `FunctionList`. Just curious

Reply:
Just a historical convention. No particular reason